app/views/govuk_publishing_components/components/docs/reorderable_list.yml

```diff
@@ -0,0 +1,76 @@
+name: Reorderable list
+description: A list of items that can be reordered
+body: |
+  List items can be reordered by drag and drop or by using the up/down buttons.
+  On small viewports the drag and drop feature is disabled to prevent being triggered
+  when scrolling on touch devices.
+
+  This component uses SortableJS - a JavaScript library for drag and drop interactions.
+  When JavaScript is disabled a set of inputs will be shown allowing users to provide
+  an order index for each item.
```

Comment:
Do you think we should document the structure of the data submitted by this component? I'm wondering if that would be a tricky thing about knowing how to use this component in an app.

I guess we could say something like:

When this component is embedded into a form and that form is submit you will receive a parameter of ordering (which can be customised with the `input_name` option). This is submitted as a collection which Rails will interpret as hash of item id => ordering position.  For example, for two items with id "a", "b" that are ordered accordingly, you'd receive a submission of `ordering[a]=1&ordering[b]=2`.

Reply:
Done